src/utils/calculateClicksAndImpressions.js

```diff
@@ -0,0 +1,14 @@
+import _ from "lodash";
+
+const calculateClicksAndImpressions = (campaignOrChannel, data, metricName) =>
```

Comment:
you can remove now all `calculate*` functions (and tests) except this one

but add tests for this one!

Reply:
caf0a6411021cf487a19fa3e55383711464635b3